R/Map_Functions.R

```diff
@@ -468,7 +468,14 @@ plot_GCAM <- function(mapdata, col = NULL, proj = robin, extent = EXTENT_WORLD,
 
     # Generate graticule (latitude/longitude lines) and clip map to extent specified.
     grat <- gen_grat()
-    mappolys <- get_bbox_polys(dataset = mapdata)
+    
+    # Only clip polygons if using an AEA projection; otherwise this can induce distortions
+    if(proj %in% c(na_aea, ch_aea)){
```

Comment:
This will not cover all Albers projections; it only gets the ones we have chosen to represent North America and China.  If this is truly an issue with the Albers projection, then we need to cover all variants of it.